src/people/widgetViews/workspace/style.ts

```diff
@@ -1366,13 +1366,20 @@ export const StoriesButtonGroup = styled.div`
 
 export const GenerateStoriesModal = styled.div`
   background: #fff;
-  width: 600px;
+  width: 800px;

```

Comment:
Please can we use percentages here rather than using an absolute value.

Like we want to use 65%.

And maybe add for smaller screens and the width could be 90%

Reply:
Sorry, I have seen your comments now 😔 
You are right we should have to be used a dynamic style approach